poetry/installation/chooser.py

```diff
@@ -0,0 +1,155 @@
+import re
+
+from typing import List
+from typing import Tuple
+
+from poetry.core.packages.package import Package
+from poetry.core.packages.utils.link import Link
+from poetry.repositories.legacy_repository import LegacyRepository
+from poetry.repositories.pypi_repository import PyPiRepository
+from poetry.utils.env import Env
+from poetry.utils.patterns import wheel_file_re
+
+
+class Wheel(object):
```

Comment:
@agronholm Yet another place a `Wheel` class has been invented. :)

Reply:
This one looks like it still supports Python 2, so the new wheel API will not help here.